### DISCUSSION TITLE
Comment justifier un texte sur le blog?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1208] [Question / réponse] [answered]
Comment faire en sorte de justifier le texte d'un article sur le blog?

<img width="1224" alt="Capture d’écran 2022-03-25 à 12 11 49" src="https://user-images.githubusercontent.com/93709421/160110924-53ea2e6a-72a4-4481-952b-39c5582207e5.png">

**Comment by alemangui** [comment, marked as answer]
Il me semble en effet que ce n'est pas possible. Par contre, en général le texte justifié est pas très conseillé pour deux raisons :

- Coté accessibilité c'est pas top. Vu que l'espace entre caractères devient variable, les personnes ayant certaines conditions trouvent plus difficile de reconnaître les mots individuels (https://www.w3.org/TR/WCAG20-TECHS/G169.html)
- Sur les petits écrans la variabilité d'espaces devient beaucoup plus important et pas très joli (point 5 ici : https://www.webdesignerdepot.com/2015/02/7-simple-rules-for-mobile-typography/)

![image](https://user-images.githubusercontent.com/1225929/160648453-1d87596b-346d-4e4e-8dc6-235043e70234.png)

**Comment by ghost** [comment]
@alemangui, c'est noté. Merci pour cette réponse très bien étayée. :-)
